spec/unit/pops/types/p_object_type_spec.rb

```diff
@@ -462,7 +458,7 @@ def parse_object(name, body_string)
         equality => [a,b]
       OBJECT
       expect { parse_object('MyObject', obj) }.to raise_error(Puppet::ParseError,
-        "equality is referencing function 'b'")
+        "MyObject equality is referencing function MyObject[b]")
```

Comment:
think it needs to say why that is wrong "function reference not allowed in equality" or somesuch.